src/main/scala/dotty/dokka/tasty/BasicSupport.scala

```diff
@@ -104,6 +104,18 @@ class SymOps[R <: Reflection](val r: R):
       symbol.is(dotc.core.Flags.Open)
     }
 
+    def hackFlagsForImplicitClasslikes: Seq[Modifier] =
+      if sym.isClassDef && (
```

Comment:
Could we pull components of this if into methods for readablity? Also if I am not mistaken it will add `implicit` to `B` here:

```
object A:
  class B
  implicit object B
```

Reply:
Yea, that was true, but fixed that. Separated conditions into methods